store/tikv/gc_worker.go

```diff
@@ -113,6 +106,20 @@ func (w *GCWorker) start() {
 	for {
 		select {
 		case <-ticker.C:
+			if w.session == nil {
+				if !w.storeIsBootstrapped() {
```

Comment:
Could we sleep for a while here? Bootstrap will do a lot of ddl. It may take a long time.